### PR TITLE
Fix bug #185

### PR DESCRIPTION
**Pull request by nicoleeeluo**
## Description
1. Commit to fix bug #185.
2. Commit to fix bug #183.
3. Comment out optional result verification cells to avoid similar issues
4. Update notebook for latest NeMo Curator version

## Usage

# Add snippet demonstrating usage
```
## Checklist
<!--
Note: All commits need to be signed and signed off. This can be done via `-sS` flags while commiting
`git commit -sS -m "...."
-->
- [ ] I am familiar with the [Contributing Guide](https://github.com/NVIDIA/NeMo-Curator/blob/main/CONTRIBUTING.md).
- [ ] New or Existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

File: tutorials/single_node_tutorial/single_gpu_tutorial.ipynb [hunk 1]
```diff
@@ -91,42 +91,48 @@
...
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
+   "id": "97079227-d9c3-40d2-b939-64b221b86990",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!export DASK_DATAFRAME__QUERY_PLANNING=False"
```
Comment (ryantwolf) [inline tutorials/single_node_tutorial/single_gpu_tutorial.ipynb:113]: This should be changed to
```
%env DASK_DATAFRAME__QUERY_PLANNING False
```

And after doing that you should be able to remove the line below too:
```python
os.environ['DASK_DATAFRAME__QUERY_PLANNING'] = 'False'
```

**Review by ryantwolf — CHANGES_REQUESTED** [review]
Thanks for doing this @nicoleeeluo! I just have one small suggestion.

File: tutorials/single_node_tutorial/single_gpu_tutorial.ipynb (hunk 1 above)
Comment (nicoleeeluo) [reply tutorials/single_node_tutorial/single_gpu_tutorial.ipynb]: @ryantwolf Hi Ryan, fix updated.

Comment (ryantwolf) [reply tutorials/single_node_tutorial/single_gpu_tutorial.ipynb]: @nicoleeeluo can you delete the code instead of commenting it out?

Comment (nicoleeeluo) [reply tutorials/single_node_tutorial/single_gpu_tutorial.ipynb]: @ryantwolf Thanks for pointing this out! I have deleted them

**Review by ryantwolf — APPROVED** [review]
Thank you!

File: .pre-commit-config.yaml
Comment (ryantwolf) [inline .pre-commit-config.yaml:1]: Ah wait I just noticed this. Can this change be removed? The PR should only modify the notebook

**Review by ryantwolf — CHANGES_REQUESTED** [review]
Oops I missed a small error. Looks like a file (`.pre-commit-config.yaml`) got added that shouldn't be.

**Pull request closed without merging**